src/test/java/com/google/sps/UserTest.java

```diff
@@ -0,0 +1,116 @@
+package com.google.sps;
+
+import com.google.sps.user.User;
+import static org.junit.Assert.assertEquals;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class UserTest {
```

Comment:
User class is a POJO class. No need for unit tests.

Reply:
I understand that these tests are not needed, but they were and they still are useful for me in case of changes like the ones requested for this CL. 
Should I delete this file, or can I leave it there if it's already written?

Reply:
Let's delete this. No need to submit un-needed code. We will have to maintain it later on.

Reply:
I deleted it.